### PR TITLE
Remove SysV init support

### PR DESCRIPTION
**Pull request by rjernst**
With the removal of support for older OSes, we no longer have any
supported systems which use SysV init. This commit removes support for
that legacy init system.

relates #51480

### CODE REVIEW

File: qa/os/centos-7/build.gradle [hunk 1]
```diff
@@ -1 +0,0 @@
-project.ext.shouldTestDocker = true
```
Comment (pugnascotia) [inline qa/os/centos-7/build.gradle]: Why has this been removed?

**Review by pugnascotia — DISMISSED** [review]
Looks fine in general, just one question.

File: qa/os/centos-7/build.gradle (hunk 1 above)
Comment (rjernst) [reply qa/os/centos-7/build.gradle]: It was unintentional, I've added it back

**Comment by rjernst** [comment]
@pugnascotia I've brought this up to date, can you take another look?

**Review by pugnascotia — APPROVED** [review]

**Pull request merged**